topics/acl.md

```diff
@@ -396,6 +413,51 @@ Example:
 In the previous section it was observed how it is possible to define commands
 ACLs based on adding/removing single commands.
 
+## Selectors
+
+Redis supports additional sets of rules that are evaluated independently of the main user permissions.
+These secondary sets of permissions are called selectors and added by wrapping a set of rules within parentheses.
+Selectors are evaluated after the main user permissions are checked and are evaluated in the order they are added to the user. 
+A command is allowed if either the root user permissions or any selectors matches a command.
```

Comment:
```suggestion
A command is allowed if either the main user permissions or any selectors matches a command.
```

Reply:
Not sure about this, seems like I should stick with one way to describe the root user permission. I would think I should update everything to main in this case.